src/GovUk.Education.ExploreEducationStatistics.Common/ViewModels/IdTitleSlugViewModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿#nullable enable
+namespace GovUk.Education.ExploreEducationStatistics.Common.ViewModels;
+
+public record IdTitleSlugViewModel : IdTitleViewModel
+{
+    public string? Slug { get; init; }
```

Comment:
Hi @mmoalin, I saw that you made a recent change, changing this from 

```csharp
public required string Slug { get; init; }
```

to

```csharp
public string? Slug { get; init; }
```

but I don't understand the reasoning behind that change. How come it's nullable now?